File: NodeJS/debug/record_video.js
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch();
  const context = await browser.newContext({
    recordVideo: { dir: 'videos/' }
  });
  const page = await context.newPage();
  await page.goto('https://example.com');
  await page.waitForTimeout(3000);
  await browser.close();
})();

File: Python/debug/record_video.py
from playwright.sync_api import sync_playwright

def main():
    with sync_playwright() as p:
        browser = p.chromium.launch()
        context = browser.new_context(record_video_dir="videos/")
        page = context.new_page()
        page.goto("https://example.com")
        page.wait_for_timeout(3000)
        browser.close()

main()
